src/common/bitset_set.h

```diff
@@ -14,6 +14,7 @@
 
 #pragma once
 #include <cstdint>
+#include <fmt/base.h>
```

Comment:
not needed because of the change in fmt_common.h below?

Reply:
@ronen-fr hi Ronen, i agree with you on that this is not necessary to get the change build. but it's necessary if we intend to ensure that this header is self-contained. because, IMHO, headers should be self-contained, so that our fellow developers are not worried about the chain effect when removing a include in a popular header. please imagine, what if we remove for instance `#include <atomic>` in `src/common/ceph_context.h` because we believe that the refcount is thread safe, and wanted to improve the performance of mutating the ref counting, but somehow, we end up editing more than two hundred source files, because they rely on `ceph_context.h` directly or indirectly for including `<atomic>` . this would be a pain, and could hinder people from reducing the header dependency.

Reply:
I think I agree in general, but in this case: fmt_common is there for this specific purpose - to include the
dot-h's and some basic declarations needed when using the fmtlib.
Anyway - not something I feel strongly about...

Reply:
i see. removed.